bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -474,11 +474,11 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
   Or one specific file will be chosen over another.
 -->
 <init_interp_attributes sim_year="1850" use_cndv=".false." use_fates=".false." lnd_tuning_mode="clm4_5_GSWP3v1"
->hgrid=0.9x1.25 maxpft=17 mask=gx1v6 use_cn=.true. use_nitrif_denitrif=.true. use_vertsoilc=.true. use_crop=.false. irrigate=.false. glc_nec=10
+>hgrid=0.9x1.25 maxpft=17 mask=gx1v7 use_cn=.true. use_nitrif_denitrif=.true. use_vertsoilc=.true. use_crop=.false. irrigate=.true. glc_nec=10
```

Comment:
Here and in the entry below, as well as in the corresponding finidat entries, note that I changed `irrigate=.false.` to `irrigate=.true.`. It appears that this was incorrect before, and that irrigate was `.true.` on the original finidat files pointed to by these configurations. See the log message for commit 7d59fb160 for details.

Reply:
Yes, this makes sense, since irrigate=T for SP cases. So it looks like it was wrong before. And you changed the mask to gx1v7, since the new files are at gx1v7.